adapters/smaato/smaato.go

```diff
@@ -436,7 +465,7 @@ func setImpForAdspace(imp *openrtb2.Imp) error {
 		return nil
 	}
 
-	return &errortypes.BadInput{Message: "Invalid MediaType. Smaato only supports Banner and Video."}
+	return nil
```

Comment:
Newly created `splitImpressionsByMediaType(...)` function and the `return nil` statements in lines 459 and 465 make this line unreachable. It doesn't affect anything and I don't have a problem with it so, if you like to fix it in this PR, ok, but I will approve your PR either way. Your code looks pretty solid.

Reply:
ok, we will address this with next PR

Reply:
Awesome @el-chuck . Thank you for addressing our feedback